javascript-timers/timeout.js

```diff
@@ -0,0 +1,5 @@
+function changeHeading(string) {
+  document.querySelector('.message').innerHTML = string;
```

Comment:
Don't use `innerHTML`, it has security concerns. You should be using `textContent` instead